scripts/preprocess_data.py

```diff
@@ -22,11 +22,13 @@ def load_data(data_path: str, tokenize: bool = False, tokenizer_type: str = "jus
     elif tokenizer_type == "spacy":
         nlp = spacy.load('en')
         tokenizer = Tokenizer(nlp.vocab)
-
     tokenized_examples = []
     with tqdm(open(data_path, "r"), desc=f"loading {data_path}") as f:
         for line in f:
-            example = json.loads(line)
+            if data_path.endswith(".jsonl") or data_path.endswith(".json"):
```

Comment:
This is making the assumption that if it is a .jsonl file, that each JSON object contains a 'text' field. I am okay with this if we mention it in the README, otherwise it may confuse people.

Reply:
yeah, let's be explicit about that in the README.